woocommerce/woocommerce-ios
# Dark Mode: update Reviews tab empty state image

Illustration update for #1555 

## Changes

- Added image PDFs from Figma to the app for Dark and Light modes
- Added a `UIImage` extension for the empty reviews image
- Used the new empty reviews image for the Reviews tab empty state

## Testing

Shortcut to show the empty state: change the OR to AND in the predicate: https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/ViewRelated/Notifications/DefaultReviewsDataSource.swift#L113

- Launch the app
- Go to Reviews tab where no reviews are expected --> the empty state image should look as expected in Dark/Light modes (note that if you switch between Light/Dark modes, the button border color might not be updated as we need to manually update it in `traitCollectionDidChange`. Let's do so in a future release)

## Example screenshots

Dark | Light
-- | --
![Simulator Screen Shot - iPhone 8 - 2019-12-12 at 00 04 08](https://user-images.githubusercontent.com/1945542/70638171-5d476c00-1c73-11ea-8322-9fe40524c00b.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-12 at 00 04 18](https://user-images.githubusercontent.com/1945542/70638175-5de00280-1c73-11ea-8319-cfce9e6777bd.png)

Update release notes:
